shell/platform/tizen/public/flutter_platform_view.h

```diff
@@ -19,8 +19,7 @@ class PlatformView {
   PlatformView(flutter::PluginRegistrar* registrar,
                int view_id,
                void* platform_window)
```

Comment:
`platform_window` can be removed from constructor.

Reply:
You're right!! Thanks! 👍